### ISSUE TITLE
WooCommerce Product Summary Block Displays Product Summary as Plain Paragraph Instead of HTML

### ISSUE THREAD

**Issue by mehediasadpur** [woothemes/woocommerce#53536]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The WooCommerce product summary block is displaying the product summary as plain text instead of rendering it as HTML. This issue prevents the inclusion of rich formatting, such as links, bold text, or line breaks, in the product summary. As a result, any HTML tags added to the product summary field are stripped out or ignored, leading to a lack of proper presentation and formatting on the product page.  

### Expected behavior

The product summary block should render HTML tags correctly, allowing for rich text formatting. 

### Actual behavior

The product summary block displays the summary as plain text, ignoring any HTML tags.  

### Steps to reproduce

1. Navigate to a WooCommerce product in the WordPress admin panel.  
2. Add HTML formatting (e.g., `<strong>`, `<a>`, or `<br>`) to the product summary field.  
3. Save the changes and view the product on the frontend.  
4. Observe that the summary appears as plain text, with no HTML formatting applied.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 9.4.3
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.8.2
Log Directory Writable: ✔
WP Version: 6.7.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: –
Language: en_US
External object cache: –

### Server Environment ###

Server Info: LiteSpeed
PHP Version: 8.1.30
PHP Post Max Size: 128 MB
PHP Time Limit: 300
PHP Max Input Vars: 1000
cURL Version: 7.61.1
OpenSSL/1.1.1k

SUHOSIN Installed: –
MySQL Version: 10.6.19-MariaDB
Max Upload Size: 128 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 32
elementor_library: 1
mailpoet_page: 1
page: 13
post: 4
product: 2388
product_variation: 2
revision: 518
shop_coupon: 2
shop_order: 9
shop_order_placehold: 32
wp_block: 3
wp_font_face: 33
wp_font_family: 12
wp_global_styles: 2
wp_navigation: 1
wp_template: 8
wp_template_part: 5
wpcode: 41
wpforms: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (17) ###

WooCommerce Patient Reports: by Your Name – 1.0
Block Visibility: by Nick Diego – 3.7.0
Capture Form Data: by Your Name – 1.0
Font Awesome: by Font Awesome – 4.5.0
Site Kit by Google: by Google – 1.141.0
WPCode Lite: by WPCode – 2.2.3.1
Jetpack: by Automattic – 14.1
Limit Login Attempts Reloaded: by Limit Login Attempts Reloaded – 2.26.16
Mailchimp for WooCommerce: by Mailchimp – 4.4.1
Nextend Social Login: by Nextendweb – 3.1.16
Spectra: by Brainstorm Force – 2.17.0
WooCommerce Bulk Order: by Your Name – 1.0
WooCommerce.com Update Manager: by Automattic – 1.0.3
WooCommerce: by Automattic – 9.4.3
Yoast SEO: by Team Yoast – 24.0
WP Mail SMTP: by WP Mail SMTP – 4.2.0
WPForms Lite: by WPForms – 1.9.2.3

### Inactive Plugins (2) ###

Tawk.to Live Chat: by Tawkto – 0.8.7
WP Call Button: by Syed Balkhi – 1.4.3

### Dropin Plugins () ###

maintenance.php: maintenance.php

### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: BDT (৳ )
Currency Position: right
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: easy_product_bundle (easy_product_bundle)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 935 KB

### WC Pages ###

Shop base: #7 - /shop/
Cart: #8 - /cart/ -  Contains the [woocommerce_cart] shortcode
Checkout: #9 - /checkout/ -  Contains the [woocommerce_checkout] shortcode
My account: #10 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Five
Version: 1.0
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ❌ Not declared

### Templates ###

Overrides: twentytwentyfive/woocommerce/content-product.php
/home/onlinedi/public_html/wp-content/plugins/woocommerce/templates/block-notices/error.php
/home/onlinedi/public_html/wp-content/plugins/woocommerce/templates/block-notices/notice.php
/home/onlinedi/public_html/wp-content/plugins/woocommerce/templates/block-notices/success.php


### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
blueprint
reactify-classic-payments-settings
use-wp-horizon

Daily Cron: ✔ Next scheduled: 2024-12-08 07:43:12 +00:00
Options: ✔
Notes: 71
Onboarding: completed

### Action Scheduler ###

Complete: 4,651
Oldest: 2024-11-07 08:57:44 +0000
Newest: 2024-12-08 04:23:13 +0000

Failed: 32
Oldest: 2024-09-03 17:34:53 +0000
Newest: 2024-12-07 12:01:03 +0000

Pending: 10
Oldest: 2024-12-08 08:59:38 +0000
Newest: 2024-12-12 22:03:55 +0000


### Status report information ###

Generated at: 2024-12-08 05:11:33 +00:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by masonsi-dev** [comment]
Same issue as yours

**Comment by imanish003** [comment]
Hi @mehediasadpur,

Thank you for reporting this issue! The good news is that it has been resolved in [this PR](https://github.com/woocommerce/woocommerce/pull/48984), which is scheduled to be included in WooCommerce 9.6.0. Along with fixing this issue, the PR also brings several improvements to the product summary block. For more details, feel free to check out the linked PR.

As this resolves the issue, I’ll go ahead and close this ticket. However, if you have any further questions or concerns, don’t hesitate to leave a comment here.

Thanks again for your valuable feedback!
